amgut/lib/data_access/env_management.py

```diff
@@ -132,8 +133,12 @@ def make_settings_table():
 def populate_test_db():
     conn = SQLConnectionHandler()
 
-    with open(POPULATE_FP) as f:
-        conn.execute(f.read())
+    with gzip.open(POPULATE_FP, 'rb') as f:
+        test_db = f.read()
+
+    command = ['psql', '-d', AMGUT_CONFIG.database]
```

Comment:
Yes, although that should probably be a cleaner error. We can set up another thing like we have in qiita, `ag clean_test` to wipe out the DB and rebuild it.